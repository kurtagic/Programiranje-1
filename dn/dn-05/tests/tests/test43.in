150
1 -768
0 -878
2 -827
1 -822
1 -773
1 -868
2 -792
1 -899
1 -836
0 -861
2 -849
1 -791
0 -871
0 -883
1 -868
2 -742
0 -902
2 -798
0 -879
2 -814
2 -810
1 -808
0 -826
2 -812
1 -742
2 -851
1 -808
2 -785
0 -882
1 -847
1 -748
2 -831
1 -816
0 -827
2 -817
2 -876
0 -852
2 -895
2 -902
0 -846
1 -765
0 -745
0 -854
0 -857
0 -836
0 -860
0 -745
2 -862
0 -859
1 -770
1 -784
1 -801
0 -765
2 -776
0 -780
0 -886
1 -781
2 -743
0 -789
0 -783
2 -797
2 -791
0 -834
1 -896
1 -796
0 -781
0 -870
2 -875
0 -840
0 -765
1 -819
2 -888
0 -776
2 -745
1 -742
0 -862
2 -805
1 -838
1 -754
2 -801
2 -875
2 -754
0 -871
2 -796
0 -757
0 -780
1 -880
1 -852
0 -787
0 -824
2 -879
1 -801
1 -870
0 -802
0 -847
1 -847
0 -836
0 -745
1 -822
0 -799
0 -829
2 -800
1 -880
0 -815
1 -825
0 -837
0 -753
0 -793
2 -831
1 -832
1 -884
1 -895
1 -769
1 -840
2 -753
0 -816
2 -747
2 -747
0 -761
2 -789
0 -785
0 -776
2 -793
2 -857
0 -801
1 -857
2 -838
2 -786
2 -862
1 -795
2 -809
2 -820
2 -813
1 -882
1 -759
2 -865
2 -899
0 -854
1 -865
1 -814
1 -767
2 -817
2 -748
1 -742
1 -856
0 -856
0 -764
1 -769
1 -744
0 -820
