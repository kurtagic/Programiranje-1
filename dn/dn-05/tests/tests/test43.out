-902: 7
-901: 7
-900: 5
-899: 7
-898: 7
-896: 4
-895: 6
-894: 2
-888: 2
-887: 2
-886: 2
-885: 2
-884: 6
-883: 2
-882: 7
-881: 3
-880: 11
-879: 5
-878: 5
-877: 2
-876: 4
-875: 8
-874: 8
-871: 11
-870: 15
-869: 11
-868: 11
-867: 10
-865: 6
-864: 2
-862: 10
-861: 10
-860: 8
-859: 8
-858: 7
-857: 13
-856: 14
-855: 14
-854: 14
-853: 14
-852: 9
-851: 9
-850: 3
-849: 3
-848: 2
-847: 9
-846: 5
-845: 5
-844: 5
-843: 1
-840: 5
-839: 1
-838: 10
-837: 10
-836: 8
-835: 8
-834: 8
-833: 7
-832: 10
-831: 8
-830: 8
-829: 5
-828: 5
-827: 5
-826: 5
-825: 8
-824: 5
-823: 5
-822: 9
-821: 5
-820: 10
-819: 10
-818: 10
-817: 10
-816: 9
-815: 7
-814: 11
-813: 9
-812: 9
-811: 2
-810: 2
-809: 10
-808: 10
-805: 2
-804: 2
-802: 11
-801: 15
-800: 15
-799: 15
-798: 15
-797: 12
-796: 12
-795: 14
-793: 11
-792: 11
-791: 9
-790: 9
-789: 3
-788: 3
-787: 7
-786: 10
-785: 10
-784: 8
-783: 8
-782: 8
-781: 6
-780: 7
-779: 7
-778: 7
-777: 7
-776: 6
-775: 6
-774: 6
-773: 6
-770: 4
-769: 8
-768: 4
-767: 4
-765: 6
-764: 7
-763: 7
-762: 7
-761: 7
-760: 1
-759: 5
-758: 1
-757: 7
-756: 7
-755: 7
-754: 7
-753: 9
-752: 9
-751: 7
-750: 7
-748: 6
-747: 6
-746: 4
-745: 15
-744: 19
-743: 15
-742: 19
-741: 2
